trino/trino_test.go

```diff
@@ -418,28 +412,27 @@ func TestTypeConversion(t *testing.T) {
 			ExpectedGoValue:            nil,
 		},
 	}
+
 	for _, tc := range testcases {
 		converter := newTypeConverter(tc.DataType)
 
 		t.Run(tc.DataType+":nil", func(t *testing.T) {
-			if _, err := converter.ConvertValue(nil); err != nil {
-				t.Fatal(err)
-			}
+			_, err := converter.ConvertValue(nil)
+			assert.NoError(t, err)
 		})
 
 		t.Run(tc.DataType+":bogus", func(t *testing.T) {
-			if _, err := converter.ConvertValue(struct{}{}); err == nil {
-				t.Fatal("bogus data scanned with no error")
-			}
+			_, err := converter.ConvertValue(struct{}{})
+			assert.Error(t, err, "bogus data scanned with no error")
 		})
+
 		t.Run(tc.DataType+":sample", func(t *testing.T) {
 			v, err := converter.ConvertValue(tc.ResponseUnmarshalledSample)
-			if err != nil {
-				t.Fatal(err)
-			}
-			if !reflect.DeepEqual(v, tc.ExpectedGoValue) {
-				t.Fatalf("unexpected data from sample:\nhave %+v\nwant %+v", v, tc.ExpectedGoValue)
-			}
+			require.NoError(t, err)
+
+			require.Truef(t,
+				reflect.DeepEqual(v, tc.ExpectedGoValue),
```

Comment:
Could you use `require.Equal()` here too? It seems to fall back to `DeepEqual` anyway: https://github.com/stretchr/testify/blob/master/assert/assertions.go#L65

Reply:
Sure thing, will do.